### ISSUE TITLE
transcript file is not being loaded - the emulator window is empty

### ISSUE THREAD

**Issue by gbelenky** [Microsoft/BotFramework-Emulator#722]
I created some transcript files from the samples but nothing happens after I open them in the emulator. The emulator window is just blank.

**Comment by tonyanziano** [comment]
Hi @gbelenky,

This is most likely due to the transcript containing some new features that are net yet supported by transcript mode in the emulator, such as conversation updates (add / remove user) and multiple users in a single conversation.

This is a known issue and is being tracked here #675.

**Comment by gbelenky** [comment]
Hi Tony, 
even that one doesn't work : 
user=vishwac
bot=cafebot

user: Who are you?
bot: Hi, I'm the Contoso Cafe bot. What's your name?
user: I'm vishwac
bot: [Typing][Delay=3000]
Hello Vishwac. Nice to meet you!

**Comment by tonyanziano** [comment]
Hey @gbelenky ,

I tried with the chat you posted and I was able to reproduce the error. I looked into the generated transcript, and it appears that there is a `conversationUpdate` message activity at the very beginning of the transcript that adds the bot in the `membersAdded` property:

```
{
    "conversation": {
      "id": "/5lS8CYG9a/MGFAqznuCdyqc0G4="
    },
    "id": "1",
    "timestamp": "2018-08-02T17:42:44.000Z",
    "type": "conversationUpdate",
    "channelId": "chatdown",
    "membersAdded": [
      {
        "id": "jopVZjYqyE3A5EnA8dYrlA+Mubw=",
        "name": "cafebot",
        "role": "bot"
      }
    ],
    "membersRemoved": []
  }
```

Previously, this was not the case and chatdown did not generate this initial activity, so I will look into what changed.

This is what is causing the issue and should be fixed once #675 is merged and makes it into the build. I deleted the aforementioned activity from the generated transcript and the emulator was able to render it just fine.
